### DISCUSSION TITLE
Usage details are not getting updated

### DISCUSSION THREAD

**Discussion by shridhar95** [Sunbird-coKreat/Community#99] [Bugs]
Hi Team,
The **usage details**  for users in co-kreat are not getting updated when the content/collection consumed by the user. As per our findings we identifed that the usage details data getting fetched from coKreat composite search which is not getting updated when the same gets updated in DIKSHA ES post content being consumed.

<img width="960" alt="image" src="https://github.com/Sunbird-coKreat/Community/assets/125978181/80491b89-5a56-4c5a-afdb-4df44c07ebcb">

**Comment by Aparna-karri591** [comment]
@amitpriyadarshi can you please look into this.

cc: @@vinukumar-vs  @PoojaJManjooran

**Comment by shridhar95** [comment]
Hi @amitpriyadarshi,
Could you please help us on updating the usage details.

**Comment by amitpriyadarshi** [comment]
@shridhar95 can you confirm which version of Sunbird Knowlg flink jobs are deployed?

**Reply by tsprasath** [reply]
@amitpriyadarshi  release-5.2.3_RC3

**Comment by amitpriyadarshi** [comment]
@tsprasath @shridhar95 these metadata gets pushed from from Sunbird ED to coKreat with the help of `metrics-data-transformer` flink job. Please validate where this job is running properly in SunbirdEd env without any error or not.

cc: @vinukumar-vs @Aparna-karri591

**Reply by tsprasath** [reply]
@amitpriyadarshi We dont have this `metrics-data-transformer` flink jobs in diksha production. 

Help me out with the deployment pipeline and ansibles values are to be added in the private repo

**Comment by amitpriyadarshi** [comment]
@tsprasath as per my understanding this job should be available in Diksha env. Please check the jenkins job `Deploy/dev/KnowledgePlatform/FlinkJobs`  for parameter `metrics-data-transformer`

cc: @vinukumar-vs

**Reply by tsprasath** [reply]
@amitpriyadarshi  Able to deploy
